benchmarking/test.py

```diff
@@ -0,0 +1,5 @@
+import daft
+
+df = daft.from_pydict({"nums": [1, 2, 3]})
+
+df.show()
```

Comment:
Note that this won't use the RayRunner because of the lack of `ray.context.set_runner_ray()` -- is that intended?

Reply:
This was accidentally committed. Removing this file.

Reply:
Removed.